pkgs/os-specific/darwin/apple-source-releases/Librpcsvc/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, appleDerivation, developer_cmds }:
+
+appleDerivation rec {
+  phases = [ "unpackPhase" "installPhase" ];
+
+  buildInputs = [ developer_cmds ];
+
+  installPhase = ''
+    export DSTROOT=$out
+    export SRCROOT=$PWD
+    export OBJROOT=$PWD
+
+    . ./xcodescripts/install_rpcsvc.sh
+
+    mv $out/usr/* $out
+    rmdir $out/usr/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "";
+    maintainers = with maintainers; [ matthewbauer ];
+    platforms   = platforms.darwin;
+    license     = licenses.apsl20;
+  };
```

Comment:
@copumpkin Is it okay if I live out the description meta? some os-specific/darwin/ have it others don't.